website/src/pages/PublicRoute.jsx

```diff
@@ -23,10 +23,15 @@ import Status from './Status/Status';
 import Error from '../utils/Error/Error';
 import Layout from '../pages/Layout'
 import Macro from './Macro/Macro';
-
-
+import Micro from './Micro/Micro';
 
 const PublicRoute = () => {
+    const test = () => {
+        const [error, setError] = useState(null);
+        const [isLoading, setIsLoading] = useState(true);
+    }
```

Comment:
Is this `test` function still relevant?

Reply:
No, it's an oversight on my part. I need to remove it :)